### DISCUSSION TITLE
Sourcing assignment

### DISCUSSION THREAD

**Discussion by EliisaE** [community/community#122201] [Other Feature Feedback, Questions, & Ideas]
Hi!

Yes, I'm totally new here and actually a recruiter wanting to focus more on tech. Got an assignment, but after trying to solve it myself, still unable to reach the answer. Maybe someone more familiar with Github, can help me :) Written assignment: Who is the most popular C++ contributor on Github in Estonia and how many own repositories does he have in public access? Please also explain how you got there.

**Comment by daniyalmaster693** [comment]
Not sure if this is the best way to do it or not, but put this in the github search bar, and then find someone that fits your criteria.

Put this in your search: location:Estonia language:C++ 
and then sort by users
